tests/components/rainmachine/test_config_flow.py

```diff
@@ -51,6 +51,39 @@ async def test_invalid_password(hass):
         assert result["errors"] == {CONF_PASSWORD: "invalid_auth"}
 
 
+async def test_options_flow(hass):
+    """Test config flow options."""
+    conf = {
+        CONF_IP_ADDRESS: "192.168.1.100",
+        CONF_PASSWORD: "password",
+        CONF_PORT: 8080,
+        CONF_SSL: True,
+    }
+
+    config_entry = MockConfigEntry(
+        domain=DOMAIN,
+        unique_id="abcde12345",
+        data=conf,
+        options={CONF_ZONE_RUN_TIME: 900},
+    )
+    config_entry.add_to_hass(hass)
+
+    with patch(
+        "homeassistant.components.rainmachine.async_setup_entry", return_value=True
+    ):
+        result = await hass.config_entries.options.async_init(config_entry.entry_id)
```

Comment:
We need to set up the integration before starting the options flow to make sure that the config flow module is loaded. Otherwise we're relying on the other tests to do that.

Reply:
The config entry is already set up above (lines 14-20); are you thinking something else?

Reply:
I don't see that. We only create a mock config entry and add it to hass. It needs to be set up with `hass.config_entries.async_setup`.